File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        map<int, int> m;
        for(auto x : nums) {
            if(m.count(x)) {
                m[x]++;
            } else {
                m.insert({x, 1});
            }
        }
        vector<pair<int, int>> ans;
        for(auto x : m) {
            ans.push_back({x.second, x.first});
        }
        sort(ans.rbegin(), ans.rend());
        vector<int> answer;
        for(int i = 0; i < k; i++) {
            answer.push_back(ans[i].second);
        }
        return answer;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        d = {}

        for i in nums:
            d[i] = True
        
        ret = 0
        for i in d: 
            k = 1
            while (i+k) in d:
                k += 1
            ret = max(ret, k)
        return ret

            
        
